kong/db/schema/others/declarative_config.lua

```diff
@@ -350,7 +350,8 @@ local function populate_references(input, known_entities, by_id, by_key, expecte
     local child_key
     if parent_entity then
       local parent_schema = all_schemas[parent_entity]
-      if parent_schema.fields[entity] then
+      local field = parent_schema.fields[entity]
```

Comment:
Should we rename `field` to `entity_field`?

Reply:
Sounds good